index.html

```diff
@@ -79,6 +79,62 @@ <h2>
   </svg>
 </div>
 
+<div class="mp-section">
+  <div>
+    <h2><a href="members/">Our members</a></h2>
+    <p>Our amazing members push us and inspire us every day. </p>
```

Comment:
This is awkward filler: it's in the plural tense, and it's devoid of content. I'd prefer to say something about how our members are engaging in a revolutionary act through their daily working practices, and we couldn't be more proud of them (but pithier!)
```
    <p>Our amazing members push us and inspire us every day. </p>
```

Reply:
How about: Our members are taking charge of their digital transformation through open collaboration with other public organizations. We're proud of their commitment!

Reply:
I don't think 'we are proud of them' communicates that our members are members and our boss :) This statement speaks on behalf of all of our members and our staff.

Perhaps something like the following, although this might have too many reserved words, but it is really what we're trying to do:

```suggestion
    <p>Our public organization members are associated to build a strong foundation for public code.</p>
```

Reply:
What about:
"Together, these public organizations are building a strong foundation for public code."